File: EasyImport.WebApi/Models/OrderItem.cs
namespace EasyImport.WebApi.Models
{
    public class OrderItem
    {
        public int Id { get; set; }

        public int OrderId { get; set; }
        public int ProductId { get; set; }
        public int SupplierId { get; set; }
        public string ProductCode { get; set; }
        public string SupplierName { get; set; }
        public string Description { get; set; }
        public string PictureUrl { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public double CartonWidth { get; set; }
        public double CartonHeight { get; set; }
        public double CartonDeepness { get; set; }
        public double NetKgs { get; set; }
        public double GrossKgs { get; set; }
        public int QtyPerCarton { get; set; }
        public double II { get; set; }
        public double PIS { get; set; }
        public double Cofins { get; set; }
        public double IPI { get; set; }

        public virtual Order Order { get; set; }
        public Product Product { get; set; }
        public Supplier Supplier { get; set; }

        //public OrderDetails OrderDetails { get; set; }
    }
}

File: EasyImport-App/src/app/_models/orderItem.ts
export interface OrderItem {
  id: number;
  productCode: string;
  description: string;
  price: number;
  quantity: number;
  pictureUrl: string;
  category: string;
  ncm: string;
  supplier: string;
  cartonDeepness: number;
  cartonHeight: number;
  cartonWidth: number;
  qtyPerCarton: number;
  ii: number;
  ipi: number;
  pis: number;
  cofins: number;
  supplierId: number;
  productId: number;
}
